src/views/CourseGradeExport.vue

```diff
@@ -315,15 +315,9 @@ export default {
     clearInterval(this.exportTimer)
   },
   created() {
-    if (this.currentUser.isTeaching || this.currentUser.isAdmin || this.currentUser.isCanvasAdmin) {
```

Comment:
Here and below, specific `currentUser.isTeaching` checks on the front end are not needed, since unauthorized server calls will return an error, which our front-end interceptor will handle as usual.